Tests/UnitTests/Core/Detector/DetectorBuilderTests.cpp

```diff
@@ -0,0 +1,80 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Detector/Detector.hpp"
+#include "Acts/Detector/DetectorBuilder.hpp"
+#include "Acts/Detector/DetectorComponents.hpp"
+#include "Acts/Detector/DetectorVolume.hpp"
+#include "Acts/Detector/PortalGenerators.hpp"
+#include "Acts/Detector/interface/IDetectorComponentBuilder.hpp"
+#include "Acts/Geometry/CuboidVolumeBounds.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Navigation/DetectorVolumeFinders.hpp"
+#include "Acts/Navigation/SurfaceCandidatesUpdators.hpp"
+#include "Acts/Utilities/Enumerate.hpp"
+
+#include <stdexcept>
+
+class CompBuilder : public Acts::Experimental::IDetectorComponentBuilder {
+ public:
+  Acts::Experimental::DetectorComponent construct(
+      const Acts::GeometryContext& gctx) const final {
+    auto bounds = std::make_unique<Acts::CuboidVolumeBounds>(10., 10., 10.);
+    // Contruct the DetectorVolume
```

Comment:
```suggestion
    // Construct the DetectorVolume
```

Reply:
Fixed, I searched for more of this typos, found 18.